### ISSUE TITLE
FPP stalled during show (latest master) - FPPMON related?

### ISSUE THREAD

**Issue by Jchuchla** [FalconChristmas/fpp#1744]
v7.x-master-417-gddec08e9 (master branch)
Just updated about a half hour before showtime tonight.

**Describe the bug and Steps to reproduce**: 
I had updated fpp to the latest on master today at around 4:15.  I had to reboot the pi after the update for FPPD to start.
The show started normally at 5:00.  at about 5:30, I noticed it was dark outside.  I saw that the sequence time was not counting.  It was on "Light of Christmas" which is the 10th item in the playlist but 4th in the main loop.  (1-6 are in the lead-in)
I restarted FPPD and it restarted successfully, and continued on the next song from where it left off.

**Expected behavior**: 
Playlist should not freeze up.

**Additional context**: 
I included the section of the fppd.log from where it started up after the update reboot, up to the line before the restart I did to get it playing again.

**Additional Attachments**
[Log Snip.txt](https://github.com/FalconChristmas/fpp/files/13836373/Log.Snip.txt)

**Comment by ghormann** [comment]
It *almost* looks FPPMON related.

2024-01-04 17:00:00.025 (1707) [Playlist] /opt/fpp/src/playlist/Playlist.cpp:739: Section position 0 is outside of section New
2024-01-04 17:12:16.869 (1770) [General] /opt/fpp/src/common.cpp:752: Error parsing JSON string in LoadJsonFromString(): 'Failed to connect to kulplights.com port 443: Connection timed out'
2024-01-04 17:12:16.869 (1770) [Plugin] src/FPPMonitorPlugin.cpp:1076: FPPMon - Could not obtain FPPMon Login Token:  Failed to connect to kulplights.com port 443: Connection timed out
2024-01-04 17:14:28.650 (1707) [General] /opt/fpp/src/common.cpp:752: Error parsing JSON string in LoadJsonFromString(): 'Failed to connect to kulplights.com port 443: Connection timed out'
2024-01-04 17:14:28.651 (1707) [Plugin] src/FPPMonitorPlugin.cpp:1076: FPPMon - Could not obtain FPPMon Login Token:  Failed to connect to kulplights.com port 443: Connection timed out
2024-01-04 17:14:28.651 (1707) [General] /opt/fpp/src/fppd.cpp:797: Shutting down main loop.
2024-01-04 17:14:28.870 (1707) [General] /opt/fpp/src/fppd.cpp:1016: Stopping channel output thread.
2024-01-04 17:14:28.971 (1707) [General] /opt/fpp/src/fppd.cpp:1020: Main Loop complete, shutting down.

**Comment by Jchuchla** [comment]
I was thinking that's a possibility as well.  But I did use FPPMON on my phone to connect to the FPP to bring up its UI to check the status and restart fppd.  So i'm not sure if that error was logged when the problem actually happened, or when I logged in to correct it.   The timing suggests the latter.   The particular sequence it stalled on was the 4th in my show.  It should have started around 5:08:57 and ended 3:43 after that. So if I'm doing my math right, that's 5:12:40 The FPPMon errors started at 5:12:17.  That was right when I noticed the dark show and logged into investigate and reset.  I guess it could go either way.

The show has not have any further problems since then.
